plugins/kiali/package.json

```diff
@@ -31,13 +31,26 @@
     "@backstage/errors": "^1.2.3",
     "@backstage/plugin-catalog-react": "^1.8.5",
     "@backstage/theme": "^0.4.3",
-    "@janus-idp/backstage-plugin-kiali-common": "1.4.1",
     "@material-ui/core": "^4.9.13",
     "@material-ui/icons": "^4.11.3",
     "@material-ui/lab": "^4.0.0-alpha.45",
     "@patternfly/patternfly": "^5.1.0",
     "@patternfly/react-charts": "^7.1.1",
-    "react-use": "^17.4.0"
+    "@patternfly/react-core": "^5.1.1",
+    "@patternfly/react-icons": "^5.1.1",
+    "axios": "^1.5.1",
+    "cytoscape": "3.15.5",
+    "deep-freeze": "0.0.1",
+    "history": "^5.3.0",
+    "json-beautify": "1.0.1",
+    "lodash": "^4.17.21",
+    "lodash-es": "^4.17.21",
```

Comment:
do we need both?

Reply:
weird, maybe yarn installed that by the prompt let me remove lodash, we used lodash-es

Reply:
Well we duplciate the libs in kiali and it seems that we pulled lodash-es worngly from our standalone let me fix it